### DISCUSSION TITLE
Smooth Font With Icons?

### DISCUSSION THREAD

**Discussion by KisonHe** [Bodmer/TFT_eSPI#1107] [Q&A - Fonts] [answered]
I want to use the material-design-icons. Take the 0xe90d for example, the processing ide seems can generate the icon correctly. 
![image](https://user-images.githubusercontent.com/49135678/112860306-4a65de80-90e6-11eb-8639-69b4bfd9c2e1.png)
Then I tried printing it by doing following
```
char icon[] = {0xe9,0x0d,0x00};
tft.loadFont("a");
tft.println(icon);
```
Sadly nothing shows up on my screen.
I tried the smooth font showing some CJK characters and it works perfectly, I wonder if it is possible to use these icons?

**Comment by KisonHe** [comment, marked as answer]
OK, I figured it out. 
I misunderstood Unicode with UTF-8. For any one might have same question,  Unicode Bytes != UTF-8 Enocding
After changing `char icon[] = {0xe9,0x0d,0x00};` to `char icon[] = {0xEE, 0xA4, 0x8D, 0x00};` it works nicely.

**Reply by baloyou** [reply]
i have a question the same,Can you provide more details? thanks.

**Comment by Bodmer** [comment]
There are a number of ways that a string or character can be represented. For example these work:
```
tft.println("€");
tft.println("\u20AC"); //  The Euro symbol is 0x20AC
```

**Reply by delhoume** [reply]
and you can use drawGlyph with an unicode code point, as you only have one char to display.
